ts/features/idpay/payment/xstate/services.ts

```diff
@@ -83,16 +93,15 @@ const createServicesImplementation = (client: IDPayClient, token: string) => {
       E.fold(
         failure => Promise.reject(failure),
         flow(
+          // eslint-disable-next-line sonarjs/no-identical-functions
```

Comment:
why not extract this to a function ? I believe it'd make it a slimmer and more stable read